modules/home-manager/tmux.nix

```diff
@@ -18,12 +19,33 @@ let
 in
 {
   options.programs.tmux.catppuccin =
-    lib.ctp.mkCatppuccinOpt "tmux";
+    ctp.mkCatppuccinOpt "tmux"
+    // {
+      plugin = mkOption {
+        type = types.package;
+        default = plugin;
+        description = "Catppuccin tmux plugin";
+      };
```

Comment:
```suggestion

```

See my comment [here](https://github.com/catppuccin/nix/pull/133#discussion_r1576653324)

tl;dr: Replacing each theme source should be done in #129, not on a per module basis